File: XWStatusVolumeManagerDome/XWStatusVolumeManagerDome/ViewController.m
//
//  ViewController.m
//  XWStatusVolumeManagerDome
//
//  Created by 邱学伟 on 2018/4/19.
//  Copyright © 2018年 邱学伟. All rights reserved.
//

#import "ViewController.h"
#import "TLStatusVolumeManager.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    [TLStatusVolumeManager regiseter:nil];
    [TLStatusVolumeManager addCustomVolumeView];
}


@end

File: XWParticleDemo-粒子效果/XWParticleDemo-粒子效果/ViewController.swift
//
//  ViewController.swift
//  XWParticleDemo-粒子效果
//
//  Created by 邱学伟 on 2016/12/30.
//  Copyright © 2016年 邱学伟. All rights reserved.
//

import UIKit

class ViewController: UIViewController,Particleable {

    override func viewDidLoad() {
        super.viewDidLoad()
        addParticleEffect(view)
    }
}

